src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig

```diff
@@ -165,21 +165,25 @@
 {%- endblock choice_widget_collapsed %}
 
 {% block choice_widget_expanded -%}
-    {% if '-inline' in label_attr.class|default('') %}
+    {% if '-inline' in label_attr.class|default('') -%}
```

Comment:
looks like the whitespace control changes are unrelated to this feature and should be done in a different PR